DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: pika
-Title: Calculates Lag and Correlation Between Two Time Series
-Version: 0.0.0.9000
+Title: A lightweight package for evaluating correlation between time series
```

Comment:
If you think the package might go on CRAN you need to confirm to their very particular rules (title case, no mention of package, no beginning with an article etc - so "Evaluate Correlation Between Time Series")

Reply:
so it would seem that the previous title would be better